tokenserver/assignment/sqlnode/sql.py

```diff
@@ -596,10 +608,16 @@ def unassign_node(self, service, node, timestamp=None, nodeid=None):
         )
         res.close()
 
-    def get_best_node(self, service):
+    def get_best_node(self, service, email):
         """Returns the 'least loaded' node currently available, increments the
         active count on that node, and decrements the slots currently available
         """
+        if self.lucky_user(email):
+            return (
+                self.settings.get("spanner_node_id", 800),
+                self.settings.get("spanner_entry", None)
+            )
+
```

Comment:
Huh, i'd figure that those users would be basically indistinguishable from new users in that we'd toss their previous state, but yeah, the would become candidates for the new system as well.

Reply:
We keep their old state around after a password reset, both so that it can be dealt with by the various background cleanup jobs, and so that we can prevent certain kinds of client misbehavior (e.g. the "you tried to use an old client_state value" errors). But I think it's fine to put them in the same bucket as new users here.